perf/roles/trex-ndr/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 
-trex_dir: /tmp/trex-core
+trex_install_dir: /opt/trex
+trex_version: 'v3.02'
```

Comment:
latest version is v3.03. Can you update?

Reply:
Updated ...